apps/fyle/tasks.py

```
@@ -727,6 +727,139 @@ def update_non_exported_expenses(data: Dict) -> None:
+    :return: None
+    """
+    try:
+        # Construct expense object from webhook data
+        expense_objects = FyleExpenses().construct_expense_object([expense_data], workspace.id)
+        
+        if expense_objects:
+            # Update the expense with new data
+            Expense.create_expense_objects(expense_objects, workspace.id, skip_update=False)
+            logger.info("Updated expense %s with latest data from webhook", expense.expense_id)
+    except Exception as e:
+        logger.exception("Error updating expense %s from webhook data: %s", expense.expense_id, e)
```

Ruff (0.13.1)

Redundant exception object included in `logging.exception` call

(TRY401)